src/new-components/text-field/text-field.test.tsx

```diff
@@ -182,4 +183,15 @@ describe('TextField', () => {
         expect(inputElement).toHaveValue('Software developer')
         expect(screen.getByTestId('value')).toHaveTextContent('Software developer')
     })
+
+    describe('a11y', () => {
+        it('renders with no a11y violations', async () => {
+            const { container } = render(
+                <TextField label="Whatʼs your name?" hint="We need it for billing purposes" />,
```

Comment:
Test it without hint, to be consistent with other field tests, and with a hint as well.